turn_off_email_notifications_for_all_users/opt_out_users_email_notifications.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+import datetime
+from typing import Optional
+
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.ingestion.graph.client import get_default_graph
+from datahub.metadata.schema_classes import CorpUserSettingsClass
+
+graph = get_default_graph()
+users = []
+
+scroll_id = None
+while True:
+    ret = graph.execute_graphql("""query scrollAcrossEntities($input: ScrollAcrossEntitiesInput!) {
```

Comment:
You can replace this with graph.get_urns_by_filter method which does the iteration for you.